sensor/common/detector/enricher.go

```diff
@@ -120,7 +123,7 @@ outer:
 	}
 }
 
-func (c *cacheValue) scanAndSet(ctx context.Context, svc v1.ImageServiceClient, req *scanImageRequest) {
+func (c *cacheValue) scanAndSet(ctx context.Context, svc v1.ImageServiceClient, req *scanImageRequest, scan *scan.LocalScan) {
```

Comment:
I think this parameter is no longer needed?

Reply:
Oops! Fixed! 😅